tests/common/schema/test_schema.py

```diff
@@ -817,13 +817,11 @@ def test_group_tables_by_resource(schema: Schema) -> None:
                 "columns": {},
                 "name": "c_products",
                 "resource": "products",
-                "write_disposition": "append",
```

Comment:
please remove changes in all the tests below. they should pass. the only problem I see is in `def apply_hints(` in `hints.py where we use `make_hints` to create new table, there you must pass `DEFAULT_WRITE_DISPOSITION` if `write_disposition` is None